html-forms/html-forms/index.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>HTML Forms</title>
+</head>
+<body>
+  <!--
+
+  1. Create h1 with Coding for
+
+  2. Create form for email requiring text content
+  3. Create a selection form for scroll down menu with Why is coding cool?
+  4. Create a form for radio buttons with the name of favorite Lanagugage (choose one) & html, css, javascript
+  5. Create a form for a check box "Would you like to sign up for more information?"
+  6. Create a form of text box for user input
+  6. Create a form for submit
+  -->
+  <h1>Coding Form!</h1>
+  <form action="" method="get" name="email" id="email">
+   <label>Email<br /></label>
```

Comment:
This `label` is missing a `for` attribute, and the value of that attribute should match the value of the `id` attribute on the element it is paired with.